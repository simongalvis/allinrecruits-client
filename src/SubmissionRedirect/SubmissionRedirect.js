import React from 'react';
import './SubmissionRedirect.css'
import NavBar from '../NavBar/NavBar'
import ApiContext from '../ApiContext';

class SubmissionRedirect extends React.Component{


    render(){
        return(
            <div className="SubmissionRedirect">
                <NavBar/>
                <h2>Thank you for your submission!</h2>
                <p>We'll process your application and let you know when we have available positions that suit your profile</p><br/>
               
    
            </div>
        )
    }
}
export default SubmissionRedirect;