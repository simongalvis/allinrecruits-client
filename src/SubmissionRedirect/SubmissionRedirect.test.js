import React from 'react'
import ReactDOM from 'react-dom'
import SubmissionRedirect from './SubmissionRedirect'
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom'

describe(`SubmissionRedirect component`, () =>{
    it(`renders without crashing`, () =>{
        const div = document.createElement('div');

        ReactDOM.render(<BrowserRouter><SubmissionRedirect/></BrowserRouter>, div)

        ReactDOM.unmountComponentAtNode(div);
    })
    it(`Renders UI as expected`, () =>{
        const tree = renderer
            .create(<BrowserRouter><SubmissionRedirect/></BrowserRouter>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})