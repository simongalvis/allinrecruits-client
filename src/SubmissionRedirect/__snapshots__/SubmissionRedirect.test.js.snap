// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmissionRedirect component Renders UI as expected 1`] = `
<div
  className="SubmissionRedirect"
>
  <div
    className="NavBar"
  >
    <header>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="all-in-education-logo"
          id="main-logo"
          src="all-in-logo.png"
        />
      </a>
      <a
        href="/admin-login"
      >
        <button
          id="admin-login-btn"
        >
          Admin Login
        </button>
      </a>
    </header>
  </div>
  <h2>
    Thank you for your submission!
  </h2>
  <p>
    We'll process your application and let you know when we have available positions that suit your profile
  </p>
  <br />
</div>
`;
